src/api/app/views/webui2/webui/user/_info.html.haml

```diff
@@ -2,65 +2,60 @@
   .card-body
     = image_tag_for(user)
 
-    %h4.home-usertitle.mt-4
-      #home-realname
-        = user.realname
-    %h4.d-inline-block#home-login
+    %h4.mb-0.mt-4#home-realname
+      = user.realname
+    %h5.d-inline-block.mb-0.align-middle.text-muted#home-login
       = user.login
+    - if User.session
+      = mail_to(user.email, title: "Send mail to #{user.name}", class: 'd-block') do
+        %i.fas.fa-envelope
+        = user.email
     - role_titles.each do |title|
-      %span.badge.badge-secondary.align-top
+      %span.badge.badge-secondary
         = title.upcase
 
-    - if User.session
-      %p
-        = mail_to user.email, title: "Send mail to #{user.name}" do
-          %i.fas.fa-envelope
-          = user.email
-
     - if groups.any?
-      Member of the #{'group'.pluralize(groups.size)}
-      %ul
-        - groups.each do |group|
-          %li
-            = link_to(group_show_path(group)) do
-              #{group.title}
-              %span.badge.badge-primary.align-top
-                #{group.tasks} #{'task'.pluralize(group.tasks)}
+      .mt-4
+        Member of the #{'group'.pluralize(groups.size)}
+        %ul
+          - groups.each do |group|
+            %li
+              = link_to(group_show_path(group)) do
+                #{group.title}
+                %span.badge.badge-primary.align-baseline
+                  #{group.tasks} #{'task'.pluralize(group.tasks)}
 
     - if is_user
-      - if user.in_beta?
-        %p
-          .badge.badge-info
-            In public beta program
-        = link_to(user_save_path(user: { login: user.login, in_beta: 0 }), method: :post) do
-          %i.fas.fa-sign-out-alt
-          Leave public beta program
-      - else
-        = link_to(user_save_path(user: { login: user.login, in_beta: 1 }), method: :post) do
-          %i.fas.fa-sign-in-alt
-          Join public beta program
-      %br
-      - if configuration.accounts_editable?(user)
-        - if account_edit_link.present?
-          = link_to account_edit_link do
-            %i.fas.fa-user-edit
-            Edit your account
+      .mt-4
+        - if user.in_beta?
+          = link_to(user_save_path(user: { login: user.login, in_beta: 0 }), method: :post) do
+            %i.fas.fa-toggle-on.text-primary
+            %span Leave public beta program
```

Comment:
You should make sure it's [obvious](https://uxplanet.org/checkbox-vs-toggle-switch-7fc6e83f10b8) what's on and what is off. I would suggest: Enable Beta Program.